app/root.tsx

```diff
@@ -79,61 +82,15 @@ export const links: LinksFunction = () => {
   ];
 };
 
-export function ErrorBoundary({ error }: { error: Error }) {
-  useEffect(() => console.error(error), [error]);
-  return (
-    <Document title="Error!">
-      <div className="container mx-auto space-y-3 my-6 bg-[#E6E6E6] rounded-lg p-10 shadow-xl">
-        <h1 className="text-5xl">Uh oh, something broke.</h1>
-        {error.message ? <p className="hidden">{error.message}</p> : <></>}
-      </div>
-    </Document>
-  );
-}
-
-export function CatchBoundary() {
-  let caught = useCatch();
-
-  let message;
-  switch (caught.status) {
-    case 404:
-      message = "Oops! Looks like you tried to visit a page that does not exist.";
-      break;
-
-    default:
-      throw new Error(caught.data || caught.statusText);
-  }
-
-  return (
-    <Document title={`${caught.status} ${caught.statusText}`}>
-      <div className="container mx-auto space-y-3 my-6 bg-[#E6E6E6] rounded-lg p-10 shadow-xl">
-        <h1 className="text-5xl">
-          {caught.status} {caught.statusText}
-        </h1>
-        <p className="text-lg text-zinc-500">{message}</p>
-      </div>
-    </Document>
-  );
-}
+export const ErrorBoundary = HighlightErrorBoundary;
```

Comment:
Is there anyway to customize the highlight error boundary so users don't see this in the event of an error?
<img width="957" alt="image" src="https://user-images.githubusercontent.com/84731260/207402641-41038eb7-7775-4bfa-be57-4e04c66c4862.png">

Reply:
Added a nicer error boundary